lib/optimizer/level-1/optimize.js

```diff
@@ -336,7 +336,7 @@ function optimizeZeroUnits(name, value) {
 }
 
 function removeQuotes(name, value) {
-  if (name == 'content' || name.indexOf('font-variation-settings') > -1 || name.indexOf('font-feature-settings') > -1 || name.indexOf('grid-') > -1) {
+  if (name == 'content' || name.indexOf('font-variation-settings') > -1 || name.indexOf('font-feature-settings') > -1 || name == "grid" || name.indexOf('grid-') > -1) {
```

Comment:
Hey @veleek - please use single quotes in "grid" as otherwise linter fails. Otherwise it looks good!